back/engines/free/email_campaigns/config/locales/en.yml

```diff
@@ -207,12 +207,14 @@ en:
       cta_accept_invitation: 'Accept your invitation'
       invitation_header: 'Your invitation is pending'
       preheader: '%{organizationName} sent you an invite to join their participation platform some days ago.'
+      invitation_expiry_message: 'This invitation expires in approximately %{expiryDays} days and %{expiryHours} hours.'
```

Comment:
Not a blocker, but just wondering whether the hours are needed here, or whether this is the best way of phrasing it to the user. Not sure when we send the reminder email, but if it's days ahead of expiry I think we could drop the hours.

Reply:
Yes, you're totally right. I was thinking we sent a reminder every 3 days, but looking more carefully at the code I think it's just once after 3 days.

Reply:
Changed as suggested.